### DISCUSSION TITLE
Soft-body?

### DISCUSSION THREAD

**Discussion by Barugon** [dimforge/rapier#442] [Q&A]
Any plans for soft-body physics?

**Comment by TuNiman1** [comment]
From my perspective, you can implement this yourself or switch to an actual game engine based on rapier plugins such as: bevy, fyrox etc...

**Reply by Barugon** [reply]
> From my perspective, you can implement this yourself or switch to an actual game engine based on rapier plugins such as: bevy, fyrox etc...

How would using Bevy or Fyrox provide soft-body physics since they both rely on Rapier?

**Reply by TuNiman1** [reply]
For the simple reason that both are not based only on rapier and are heavier and more powerful, I suggest you to check it out cause i actually don't really know, but it might work

**Reply by Barugon** [reply]
While Bevy uses Rapier as a plugin, Fyrox _only_ has Rapier for it's physics and neither have any soft-body capability whatsoever, which is why I asked this question.

Saying "no, we have no plans for soft-body physics in Rapier" is also a viable answer.

**Reply by TuNiman1** [reply]
I suggested mate, do what you think(:
